plugins/filter_aws/aws.c

```diff
@@ -359,6 +384,11 @@ static int get_vpc_metadata(struct flb_filter_aws *ctx)
      *latest/meta-data/network/interfaces/macs/{mac_id}/vpc-id/"
      */
     flb_sds_t vpc_path = flb_sds_create_size(70);
+    if (!vpc_path) {
+        flb_errno();
+        flb_sds_destroy(mac_id);
+        return -1;
+    }
     vpc_path = flb_sds_printf(&vpc_path, "%s/%s/%s/",
```

Comment:
Thanks for fixing the alloc check above! This call here also needs one.

Reply:
Done, https://github.com/fluent/fluent-bit/pull/6065/commits/7512737759a2203bbb2664f0c42fcdf08057def1